ArduPlane/commands_logic.cpp

```diff
@@ -34,12 +34,10 @@ bool Plane::start_command(const AP_Mission::Mission_Command& cmd)
         const uint16_t next_index = mission.get_current_nav_index() + 1;
         auto_state.wp_is_land_approach = mission.get_next_nav_cmd(next_index, next_nav_cmd) && (next_nav_cmd.id == MAV_CMD_NAV_LAND) &&
             !quadplane.is_vtol_land(next_nav_cmd.id);
-
-        gcs().send_text(MAV_SEVERITY_INFO, "Executing nav command ID #%i",cmd.id);
-    } else {
-        gcs().send_text(MAV_SEVERITY_INFO, "Executing command ID #%i",cmd.id);
     }
 
+    gcs().send_text(MAV_SEVERITY_INFO, "Executing %s(ID=%i)", cmd.type(), cmd.id);
```

Comment:
If we have human names we can drop the command ID stuff which vastly improves the end user readability, without impacting anything on the log analysis side. We have 2 fields with identical information, one of which makes no sense to a user, and most developers still have to look up the XML doc for the actual messages that an ID refers to.

Reply:
Yep.  I left the number in there as being completely unambiguous.

I'm currently heavily leaning towards getting rid of the Peter-generated strings and going to strictly "MAV_CMD_NAV_WAYPOINT" -> "NAV_WAYPOINT" etc - that will make the strings unambiguous and when I do get to making the mavlink generator changes to create these strings there won't be any change apart from a whole bunch of `-` lines in the control-logic files.  Thoughts?

Reply:
I'd rather have human strings. Status texts get shown in a message/console to the user, and presenting the more humane name seems valuable to users. I guess I could do a replacement on the string to correct it when I get it but I don't like doing that in general.

My overall preference for statustexts is that we should move them to being as generally understandable as possible for users if they are anything other then `MAV_SEVERITY_DEBUG` and that means not reflecting the mavlink name, but the name of the waypoint that their mission editor probably showed them when they added it. (To be clear this is actually a print I'd be fine with not seeing in the GCS, but if we are going to show it (which I do see the value of) then I'd like it to be a user friendly as possible)